src/collective/sidebar/tests/test_utils.py

```diff
@@ -20,3 +21,29 @@ def test_crop(self):
         # self.assertEqual(crop(u'12345678910', 5), u'12345...')
         # self.assertEqual(crop(u'.sonderzeichen:,;', 17), u'sonderzeichen...')  # noqa
         # self.assertEqual(crop(u'This should be:cropping', 20), u'This should be...')  # noqa
+
+    def test_get_icon(self):
+        from collective.sidebar.utils import get_icon
+        self.assertEqual(get_icon('__prefix__'), 'glyphicon')
+        self.assertEqual(get_icon('_unknown_icon_'), 'glyphicon _unknown_icon_')  # noqa
+        self.assertEqual(get_icon('cut'), 'glyphicon glyphicon-scissors')
+
+        api.portal.set_registry_record(name='collective.sidebar.icon_font', value='Fontello')  # noqa
+        self.assertEqual(get_icon('__prefix__'), 'icon')
+        self.assertEqual(get_icon('_unknown_icon_'), 'icon _unknown_icon_')
+        self.assertEqual(get_icon('cut'), 'icon cut')
+
+        api.portal.set_registry_record(name='collective.sidebar.icon_font', value='Font Awesome')  # noqa
```

Comment:
Combine this with Font Awesome Pro as `fap` is foobar and the prefix `fa` is globally used for both the Standard and Pro editions.

Reply:
Closing this as discussed via skype...